### PR TITLE
Fix exit()

### PR DESCRIPTION
**Pull request by ChJR**
After #1343, RustPython could not provide builtin exit() properly.

### CODE REVIEW

File: tests/snippets/exit.py [hunk 1]
```diff
@@ -0,0 +1,4 @@
+from testutils import assert_raises
```
Comment (youknowone) [inline tests/snippets/exit.py]: by convention, `tests/snippets/builtin_exit.py` looks better name of this test file.

Comment (ChJR) [reply tests/snippets/exit.py]: I think this file(exit.py) can hold sys.exit() test. Would be better if separating them?

**Comment by youknowone** [comment]
I have no idea how this breaking is related to #1343 

But the patch looks legit to me

**Comment by coolreader18** [comment]
This looks good, thanks! Could you edit `handle_exception` in `main.rs` to check if the exception is an instance of `SystemExit` and if so exit with the appropriate error code/print the args?

**Comment by ChJR** [comment]
@coolreader18 

> I have no idea how this breaking is related to #1343
> 
> But the patch looks legit to me

**Comment by ChJR** [comment]
I thought builtin exit() and sys.exit() need to placed in separate PRs. But they are related each other. I would handle them in this PR.

**Pull request merged**